Adyen/UI/Form/Items/Address/FormAddressItem.swift

```diff
@@ -164,11 +183,18 @@ public final class FormAddressItem: FormValueItem<PostalAddress, AddressStyle>,
     }
     
     private func createPickerItem(from viewModel: AddressViewModel, subRegions: [Region]) -> FormItem {
-        let defaultRegion = subRegions.first { $0.identifier == value.stateOrProvince } ?? subRegions[0]
-        let item = FormRegionPickerItem(preselectedValue: defaultRegion,
-                                        selectableValues: subRegions,
-                                        style: style.textField)
-        item.title = viewModel.labels[.stateOrProvince].map { localizedString($0, configuration.localizationParameters) }
+        let defaultRegion = subRegions.first { $0.identifier == value.stateOrProvince }
+        let itemTitle = viewModel.labels[.stateOrProvince].map { localizedString($0, configuration.localizationParameters) } ?? ""
+        let item = FormRegionPickerItem(
+            preselectedRegion: defaultRegion,
+            selectableRegions: subRegions,
+            shouldShowCountryFlags: false,
+            validationFailureMessage: localizedString(.pickerValidationFailureMessageInvalid, configuration.localizationParameters, itemTitle),
```

Comment:
```suggestion
        let itemTitle = viewModel.labels[.stateOrProvince].map { localizedString($0, configuration.localizationParameters) } ?? ""
        let failureMessage = localizedString(.pickerValidationFailureMessageInvalid, configuration.localizationParameters, itemTitle)
        let item = FormRegionPickerItem(
            preselectedRegion: defaultRegion,
            selectableRegions: subRegions,
            shouldShowCountryFlags: false,
            validationFailureMessage: failureMessage,
```

Reply:
Fixed in 54d57629db756b0dcba365a6f4b17535ea16d960